### PR TITLE
chore(release): 3.0.0

### PR DESCRIPTION
**Pull request by frederickfogerty**
This PR is a draft release for v3.0.0. I'm opening this as a PR so there is an opportunity for feedback for the release notes from @rahulk94

### CODE REVIEW

**Review by rahulk94 — COMMENTED** [review]
One suggestion for changelog

Besides that I think we can then look at fixing up anything else in a subsequent release as this should be enough to give us embroider@0.48.1 compatibility 🎉

**Comment by rahulk94** [comment]
Also raised https://github.com/imgix/ember-cli-imgix/pull/216 as I missed updating docs earlier

**Comment by frederickfogerty** [comment]
> Question @frederickfogerty do you want to change this repo to use `@imgix/js-core` during this release? 

Yeah, I think this is a great idea. Technically it wouldn't _have_ to be a breaking change since the APIs are the same, but I think it's good to get it in with this update. I'm working off your PR to try get something working 👍

**Comment by frederickfogerty** [comment]
@rahulk94 so I've merged the @imgix/js-core update in, thanks for doing most of the work on that. Do you think you could just briefly pull this project down and check everything works in one of your local projects? All the tests pass but I just want to check in a real project. After that we should be good to go!

**Review by sherwinski — APPROVED** [review]
This is minor but I wanted to call this out: let's be sure to squash any commits in this branch before merging so that all the changes get tagged under the same `chore` commit for this release.

**Comment by rahulk94** [comment]
Hey @frederickfogerty I've done a quick test and current master (2d5899ceefe4da98ab1e2f3d4622c3b5a0d7316e) looks good for normal Embroider builds but is throwing some errors when run in Fastboot (serverside render). I'll poke around a bit more about this on Tuesday and let you know to make sure this isn't an issue with this library itself.

**Comment by rahulk94** [comment]
Also we should loosen our `@embroider/macros` dependency to `>=^0.47.2` so that consumers can better manage embrdoider dependencies

**Comment by frederickfogerty** [comment]
Alright @rahulk94 I look forward to your updates about Fastboot. Let me know if I can help out in any way

**Comment by rahulk94** [comment]
@frederickfogerty sorry dude, hectic week. Haven't had a chance to test this yet. Will update you once I've had a chance to try it 🤞🏾

**Comment by rahulk94** [comment]
@frederickfogerty I managed to suss things on my end and tested against our app with `embroider` dependencies from version `0.48.1`. This version is latest and is needed to make sure `ember-get-config` works correctly.
```
    // Flags for Embroider optimizations
    staticAddonTestSupportTrees: true,
    staticAddonTrees: true,
    staticHelpers: true,
    staticModifiers: true,
```

We've got other issues atm which prevent us from `staticComponents` and `staticRoutes`.

I reckon we're good to ship this and can address any other issues in any subsequent service packs.

**Review by rahulk94 — APPROVED** [review]
This looks good to me @frederickfogerty. We can improve on this further as needed.

**Comment by rahulk94** [comment]
One last thing actually, I've tried running the `embroiderSafe` and `embroiderOptimized` options in the ember try config and both options are passing. I reckon we add this in before releasing too since its just internals/testing config now? Raised https://github.com/imgix/ember-cli-imgix/pull/220

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+### BREAKING CHANGES
+
+* The deprecated imgix-image-wrapped component has been removed.
+* Ember 2.x is no longer supported.
+* Upgraded ember-cli-babel to 7.26.6
+
+### Internal
+* Update Ember CLI blueprint to 3.28
+* Widen range of allowed ember-get-config versions
```
Comment (rahulk94) [inline CHANGELOG.md:16]:
```suggestion
* Widen range of allowed ember-get-config versions
* Add Embroider safe and optimized options to Ember try config
```

**Comment by frederickfogerty** [comment]
I'm going to merge this in now, and let's get that embroider build change in later if we can get it to work

**Comment by frederickfogerty** [comment]
Released on npm. From all of us at the SDK team at imgix, a big thanks for all the work you've put in here @rahulk94 🙌

**Pull request merged**